automated/linux/pibridge/test-pb-1.yaml

```diff
@@ -0,0 +1,19 @@
+metadata:
+    name: Test eth-ethtool
```

Comment:
Please change name to match the test goal. ETH is not tested here.

Reply:
Thanks... 
I updated it and it runs in MY device...
But this will not run in the testrack01...  We don't have the same constellation and that is not ok...
We have to talk about it before I push this test again...